tensorflow/go/genop/generate.sh

```diff
@@ -24,6 +24,15 @@ then
   GOPATH=$(go env GOPATH)
 fi
 
+# Check if it is running in git-bash
+ls -la / | grep git-bash.exe
+if [ $? -e "0" ]
+then
+  GOPATH=${GOPATH//\\/\/}
+  GOPATH=/${GOPATH//:/}
+fi
+
+
 cd $(dirname $0)
 for g in $(echo "${GOPATH//:/ }"); do
```

Comment:
Because it will convert "D:\path" to "D \path", This caused the new path to lose the drive letter.
And if making the GOPATH env variable use UNIX-style path, go generate will report 
> go: GOPATH entry is relative; must be absolute path: "/E/go-path/".

Reply:
Can you document this better with examples to illustrate what exactly this block of code this? It will help future maintainers who are not otherwise familiar with the bash replace string rules.

Reply:
Example: GOPATH=E:\go-repo
1. "${GOPATH//\\/\/}" change "E:\go-repo" to "E:/go-repo".
2. "/${GOPATH//:/}" change "E:/go-repo" to "/E/go-repo".
Can you understand this explanation? Or add corresponding annotation in the code?

Actually, I am not very familiar with the bash replace string rules.

Reply:
Yah adding this explanation to the code will be great! Thanks.